src/general/interpolation.jl

```diff
@@ -542,10 +545,11 @@ end
 
             pos_diff = point_coords - coords
             distance2 = dot(pos_diff, pos_diff)
-            pos_diff, distance2 = PointNeighbors.compute_periodic_distance(pos_diff,
-                                                                           distance2,
-                                                                           search_radius,
-                                                                           periodic_box)
+            pos_diff,
+            distance2 = PointNeighbors.compute_periodic_distance(pos_diff,
+                                                                 distance2,
+                                                                 search_radius,
+                                                                 periodic_box)
```

Comment:
```suggestion
            distance2 = PointNeighbors.compute_periodic_distance(pos_diff, distance2,
                                                                 search_radius, periodic_box)
```
This can probably fit now?